mlflow/server/js/src/model-registry/components/ModelVersionView.test.js

```diff
@@ -143,14 +176,87 @@ describe('ModelVersionView', () => {
       const deleteMenuItem = wrapper.find('.delete').hostNodes();
       expect(deleteMenuItem.prop('aria-disabled')).toBeUndefined();
       deleteMenuItem.simulate('click');
-      expect(wrapper.find(ModelVersionView).instance().state.isDeleteModalVisible).toBe(true);
+      expect(wrapper.find(ModelVersionViewImpl).instance().state.isDeleteModalVisible).toBe(true);
     }
   });
 
+  test('run link renders if set', () => {
+    const runLink =
+      'https://other.mlflow.hosted.instance.com/experiments/18722387/' +
+      'runs/d2c09dbd056c4d9c9289b854f491be10';
+    const modelVersion = mockModelVersionDetailed(
+      'Model A',
+      1,
+      Stages.NONE,
+      ModelVersionStatus.READY,
+      [],
+      runLink,
+    );
+    const runId = 'somerunid';
+    const experimentId = 'experiment_id';
+    const runInfo = mockRunInfo(runId, experimentId);
+    const expectedRunDisplayName = Utils.getRunDisplayName({}, runId);
+    const props = {
+      ...minimalProps,
+      modelVersion: modelVersion,
+      runInfo: runInfo,
+      runDisplayName: expectedRunDisplayName,
+    };
+    wrapper = mount(
+      <Provider store={minimalStore}>
+        <BrowserRouter>
+          <ModelVersionView {...props} />
```

Comment:
Should this be `ModelVersionViewImpl`? Somehow, my tests still passed when I did my PR

Reply:
I think `ModelVersionView` makes more sense here because we use `Provider` to provide states. Without `Provider`, `ModelVersionViewImpl` would make more sense.